src/disc/stk/Albany_STK3DPointStruct.cpp

```diff
@@ -12,17 +12,19 @@
 Albany::STK3DPointStruct::STK3DPointStruct(const Teuchos::RCP<Teuchos::ParameterList>& params,
                                            const Teuchos::RCP<const Teuchos_Comm>& commT,
 					   const int numParams) :
-  GenericSTKMeshStruct(params, Teuchos::null, 3, numParams)
+  GenericSTKMeshStruct(params, 3, numParams)
 {
-  partVec[0] = &metaData->declare_part("Block0", stk::topology::ELEMENT_RANK);
+  std::cout << "---3DPoint constructor---" << std::endl;
+  partVec.push_back(&metaData->declare_part_with_topology("Block0", stk::topology::PARTICLE));
```

Comment:
I don't understand why the topology would be PARTICLE. Isn't particle a 0-d topology?

Reply:
Interesting question, I do not know the answer. I think @ikalash might have been the last to touch this operation in the code before I did. Do you know Irina?

I made the change here to combine the part declaration and topology declaration (on line 22 of the main branch) in a single call instead of using two stk::mesh calls as was past practice. This change made it easier for me to pass this information to the STKConnManager (and it simplifies/robustifies these operations in the code).